test.ts

```diff
@@ -39,22 +72,61 @@ describe('does parsing', () => {
 });
 
 describe('supported value types', () => {
+  // TODO: ？？？
   it('boolean without following argument', () => {
-
+    const [options] = parse(
+      ['--yesterday=one', '-q', '9'],
+      {
+        'yesterday': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'string'
+        },
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'number'
+        }
+      }
+    );
+    expect(options).toEqual({
+      yesterday: 'one',
+      quiet: 9
+    });
   });
-
+  // TODO: error "noQuiet": undefined
+  // if use "noquiet" is ok
   it("boolean with prefixed 'no'", () => {
-
+    const [options] = parse(
+      ['--noYesterday', '-noQuiet', 'true'],
```

Comment:
This test will not pass, because I haven't implement it. 😄
Prefix no means "--no-yesterday"